### DISCUSSION TITLE
自定义识别词 的功能 搭配上了共享识别词 插件后，识别词内容太多，自定义修改了识别词后，点击保存不方便，是否有必要改进一下保存按钮的显示方式和位置

### DISCUSSION THREAD

**Discussion by MidnightShake** [jxxghp/MoviePilot#2648] [建议]
自定义识别词 的功能 搭配上了共享识别词 插件后，识别词内容太多，自定义修改了识别词后，点击保存不方便，是否有必要改进一下保存按钮的显示方式和位置

**Comment by MoogleXX** [comment]
对。自定义识别词列表太长了。保存按钮又在最后。如果在文件的最后添加自己自定义的。会被在线同步的覆盖掉。只能添加在整个最前面才不会被在线同步内容覆盖。我目前是用浏览器的鼠标手势，一键滑动到网页末端或者顶端修改。建议直接在网页顶端加个保存按钮方便一点。
